metaflow/datatools/s3.py

```diff
@@ -604,17 +780,24 @@ def _head():
     def get(self, key=None, return_missing=False, return_info=True):
         """
         Get a single object from S3.
-        Args:
-            key: (optional) a suffix identifying the object. Can also be
-                 an object containing the properties `key`, `offset` and
-                 `length` to specify a range query. `S3GetObject` is such an object.
-            return_missing: (optional, default False) if set to True, do
-                            not raise an exception for a missing key but
-                            return it as an S3Object with .exists == False.
-            return_info: (optional, default True) if set to True, fetch the
-                         content-type and user metadata associated with the object.
-        Returns:
-            an S3Object corresponding to the object requested.
+
+        Parameters
+        ----------
+        key : str or `S3GetObject`
+            Object to download. It can be an S3 url, a path suffix, or
+            an `S3GetObject` that defines a range of data to download.
+        return_missing : bool
+            If set to True, do not raise an exception for a missing key but
+            return it as an `S3Object` with `.exists == False` (default: False).
+        return_info : bool
+            If set to True, fetch the content-type and user metadata associated
+            with the object at no extra cost, included for symmetry with `get_many`
```

Comment:
I'll have to check the "no extra cost". There is a small cost in some cases, not sure which call though off the top of my head.